tests/data/GlyphsFileFormatv3.diff

```diff
@@ -0,0 +1,1676 @@
+  lastChange = "2022-07-25 16:25:56 +0000";
+  layers = (
+  {
+  anchors = (
+  {
+  name = _top;
+  pos = (107,500);
+  },
+  {
+  name = top;
+  pos = (107,728);
+  }
+  );
+  layerId = m01;
+  metricLeft = "=A";
+  metricWidth = "=A";
+  shapes = (
+  {
+  closed = 1;
+  nodes = (
+  (70,665,o),
+  (82,678,o),
+  (82,700,cs),
+  (82,718,o),
+  (70,728,o),
+  (45,728,cs),
+  (22,728,o),
+  (10,718,o),
+  (10,700,cs),
+  (10,678,o),
+  (22,665,o),
+  (45,665,cs)
+  );
+  },
+  {
+  closed = 1;
+  nodes = (
+  (195,665,o),
+  (205,678,o),
+  (205,700,cs),
+  (205,718,o),
+  (195,728,o),
+  (175,728,cs),
+  (155,728,o),
+  (142,718,o),
+  (142,700,cs),
+  (142,678,o),
+  (155,665,o),
+  (175,665,cs)
+  );
+  }
+  );
+  width = 459;
+  },
+  {
+  anchors = (
+  {
+  name = _top;
+  pos = (290,500);
+  },
+  {
+  name = top;
+  pos = (290,728);
+  }
+  );
+  layerId = "C2ECF50A-02EF-4989-A14C-AF8E838D1105";
+  metricRight = "=|";
+  shapes = (
+  {
+  closed = 1;
+  nodes = (
+  (230,573,o),
+  (263,608,o),
+  (263,650,cs),
+  (263,695,o),
+  (230,728,o),
+  (185,728,cs),
+  (143,728,o),
+  (108,695,o),
+  (108,650,cs),
+  (108,608,o),
+  (143,573,o),
+  (185,573,cs)
+  );
+  },
+  {
+  closed = 1;
+  nodes = (
+  (423,573,o),
+  (458,608,o),
+  (458,650,cs),
+  (458,693,o),
+  (423,728,o),
+  (380,728,cs),
+  (338,728,o),
+  (303,693,o),
+  (303,650,cs),
+  (303,608,o),
+  (338,573,o),
+  (380,573,cs)
+  );
+  }
+  );
+  width = 566;
+  }
+  );
+  unicode = 776;
+  },
+  {
+  export = 0;
+  glyphname = _part.test;
+  lastChange = "2023-02-25 14:14:38 +0000";
+  layers = (
+  {
+  layerId = m01;
+  partSelection = {
+  Width = 1;
+  };
+  shapes = (
+  {
+  closed = 1;
+  nodes = (
+  (5,0,l),
+  (138,0,l),
+  (38,700,l),
+  (5,700,l)
+  );
+  }
+  );
+  width = 100;
+  },
+  {
+  associatedMasterId = m01;
+  layerId = "BEE6AB80-1627-43EB-904A-37A721971B03";
+  name = Wide;
+  partSelection = {
+  Width = 2;
+  };
+  shapes = (
+  {
+  closed = 1;
+  nodes = (
+  (5,0,l),
+  (988,0,l),
+  (38,700,l),
+  (5,700,l)
+  );
+  }
+  );
+  width = 1000;
+  },
+  {
+  layerId = "C2ECF50A-02EF-4989-A14C-AF8E838D1105";
+  partSelection = {
+  Width = 1;
+  };
+  shapes = (
+  {
+  closed = 1;
+  nodes = (
+  (5,0,l),
+  (138,0,l),
+  (38,700,l),
+  (5,700,l)
+  );
+  }
+  );
+  width = 100;
+  },
+  {
+  associatedMasterId = "C2ECF50A-02EF-4989-A14C-AF8E838D1105";
+  layerId = "A4E932DD-1E0B-4399-B784-65C6F564CD7C";
+  name = Wide;
+  partSelection = {
+  Width = 2;
+  };
+  shapes = (
+  {
+  closed = 1;
+  nodes = (
+  (5,0,l),
+  (988,0,l),
+  (38,700,l),
+  (5,700,l)
+  );
+  }
+  );
+  width = 1000;
+  }
+  );
+  partsSettings = (
+  {
+  bottomValue = 0;
+  name = Width;
+  topValue = 100;
+  }
+  );
+  },
+  {
+  category = Icon;
+  glyphname = Smily;
+  lastChange = "2023-02-25 15:00:15 +0000";
+  layers = (
+  {
+  attr = {
+  color = 1;
+  };
+  layerId = m01;
+  shapes = (
+  {
+  attr = {
+- fillColor = (255,211,55,255);
++ fillColor = (
```

Comment:
Another instance deviating from the spec.

Reply:
> deviating from the spec

which "spec"?

and can you please clarify what the minus and the plus lines are supposed to represent in the diff?

Reply:
Exactly. glyphsLib does not follow the description about colors and tuples in the spec:

https://github.com/schriftgestalt/GlyphsSDK/blob/8639eaf79645e664be06e001fe0417cc74689da7/GlyphsFileFormat/GlyphsFileFormatv3.md

Reply:
> and can you please clarify what the minus and the plus lines are supposed to represent in the diff?

Sure! The red chunks (the ones with minuses) are what removed by glyphsLib relative to GlyphsFileFormatv3.glyphs, and the green ones (the ones with pluses) are what has been added by glyphsLib.